drafts/draft-ietf-ippm-ioam-ipv6-options.xml

```diff
@@ -320,12 +320,23 @@
             due to the high complexity in identifying the source of the leak.
             Such a troubleshooting process might require coordination between
             multiple operators, complex configuration verification,
-            packet capture analysis, etc.</t>
+            packet capture analysis, etc. This requirement may require 
+            additional option or fields to be defined to identify the domain
+            that inserted the IOAM data,  this is not addressed
+            in this document.</t>
```

Comment:
s/,this is not addressed in this document/This is out of the scope of this document./